pkg/models/models.go

```diff
@@ -22,6 +22,7 @@ const (
 	EcosystemPackagist = "Packagist"
 	EcosystemHex       = "Hex"
 	EcosystemPub       = "Pub"
+	EcosystemCyDxSBOM  = "CydxSbom"
```

Comment:
Uh oh. This is a problem. Ecosystem is very specific to package ecosystem (e.g. PyPI, Maven etc.) We probably need to re-think how to fit in an SBOM into an ecosystem.

Lets go ahead with `CydxSbom` ecosystem for now. We may have to refactor this in future.